conbench/templates/compare-entity.html

```diff
@@ -71,7 +64,13 @@
         <ul class="list-group">
           <li class="list-group-item active">Baseline</li>
           <li class="list-group-item" style="overflow-y: auto;">
-            <b>run</b>
+            <b>result ID</b>
+            <div align="right" style="display:inline-block; float: right;">
+              <a href="{{ url_for('app.benchmark-result', benchmark_result_id=baseline.id) }}">{{ baseline.id }}</a>
+            </div>
+          </li>
+          <li class="list-group-item" style="overflow-y: auto;">
+            <b>submitted as part of CI run</b>
```

Comment:
In practice, this is true of all our results. In theory, anyone could run benchmarks manually on a machine and submit the results to Conbench though, and we wouldn't be able to tell the difference. Not sure how useful it is to get pedantic, but maybe it's worth a second of thought.

Reply:
Oh let me forward this to #1331.